x/stake/client/cli/flags.go

```diff
@@ -37,10 +37,10 @@ func init() {
 	fsAmount.String(FlagAmount, "1steak", "Amount of coins to bond")
 	fsShares.String(FlagSharesAmount, "", "Amount of source-shares to either unbond or redelegate as a positive integer or decimal")
 	fsShares.String(FlagSharesPercent, "", "Percent of source-shares to either unbond or redelegate as a positive integer or decimal >0 and <=1")
-	fsDescription.String(FlagMoniker, "[do-not-modify]", "validator name")
-	fsDescription.String(FlagIdentity, "[do-not-modify]", "optional identity signature (ex. UPort or Keybase)")
-	fsDescription.String(FlagWebsite, "[do-not-modify]", "optional website")
-	fsDescription.String(FlagDetails, "[do-not-modify]", "optional details")
+	fsDescription.String(FlagMoniker, "", "validator name")
+	fsDescription.String(FlagIdentity, "", "optional identity signature (ex. UPort or Keybase)")
+	fsDescription.String(FlagWebsite, "", "optional website")
+	fsDescription.String(FlagDetails, "", "optional details")
```

Comment:
addressed

Reply:
Wait, but if you default it to [do-not-modify], then how to I reset a field to empty using the cli?

Reply:
The default for validator creation is an empty string "", the default for the flags for edit-candidacy is `[do-not-modify]` which just means that if you do not include the flag, it will not edit the original field you had in there. If you want to reset a field to the empty string then something like `gaiacli stake edit-validator --moniker=""` should work